packages/common/refresh-utils.ts

```diff
@@ -70,7 +71,8 @@ if (import.meta.hot && !inWebWorker) {
   }
 
   const sharedHead = removeLineBreaksIfNeeded(
-    `import * as RefreshRuntime from "${runtimePublicPath}";
+    `import * as RefreshRuntime from "${reactRefreshHost}${runtimePublicPath}";
```

Comment:
Here look like has a issue if enable vite `base` option, the vite will rewrite `/@react-refresh` to `/base/@react-refresh` at dev transformer. The host the `reactRefreshHost` is ``, so it is correct, but the remote here need to a path like `${host_url}${base}`, eg `http://localhost:3000/base`. 

I think it need to add to document.

Reply:
Good callout @underfin added.